dev/bots/analyze_sample_code.dart

```diff
@@ -19,7 +19,7 @@ import 'package:path/path.dart' as path;
 import 'package:watcher/watcher.dart';
 
 // If you update this version, also update it in dev/bots/docs.sh
-const String _snippetsActivateVersion = '0.2.3';
+const String _snippetsActivateVersion = '0.2.5';
```

Comment:
(not important for this PR but we should consider putting this in a file that both of them just read)

Reply:
Yeah, that makes sense.